genie-agent/src/main/resources/META-INF/spring.factories

```diff
@@ -1,13 +1,13 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
   com.netflix.genie.agent.cli.CliAutoConfiguration,\
   com.netflix.genie.agent.cli.JCommanderAutoConfiguration,\
-  com.netflix.genie.agent.configs.AgentAutoConfiguration,\
-  com.netflix.genie.agent.configs.ProcessAutoConfiguration,\
   com.netflix.genie.agent.execution.ExecutionAutoConfiguration,\
   com.netflix.genie.agent.execution.services.impl.grpc.GRpcServicesAutoConfiguration,\
   com.netflix.genie.agent.execution.services.impl.ServicesAutoConfiguration,\
   com.netflix.genie.agent.execution.statemachine.StateMachineAutoConfiguration,\
   com.netflix.genie.agent.execution.statemachine.actions.StateMachineActionsAutoConfiguration,\
-  com.netflix.genie.agent.rpc.GRpcAutoConfiguration
+  com.netflix.genie.agent.rpc.GRpcAutoConfiguration,\
```

Comment:
All the classes mentioned in this file are Spring autoconfigurations. Why move just some of them?

Reply:
I moved the two that were in the original `spring` package. The rest are sprinkled throughout the codebase due the classes they're creating all being package protected. Moving all of them would be a big change. I'd like to do it so that it's consistent but didn't feel like taking it on right now.